apps/aeutils/src/aeu_validation.erl

```diff
@@ -5,7 +5,7 @@
 run([], _Arg) ->
     ok;
 run([F | Rest], Arg) ->
-    case F(Arg) of
+    case apply(F, Arg) of
```

Comment:
To be a bit more explicit here Arg should be ArgList or something as I guess there are more than one argument since you need apply. (And the validation functions seem to take three arguments. Perhaps even be even more explicit and expect Arg to be a three element list and use F(Arg1, Arg2, Arg3) or even named arguments.